### ISSUE TITLE
Rocket glitches out landing on Ceres with launch controller

### ISSUE THREAD

**Issue by Fapiko** [BlesseNtumble/GalaxySpace#162]
As soon as it enters Ceres it glitches out here at Y=800 eating up fuel. Launch controller has worked fine on every other planet I've tried so far (Moon, Mars, Phobos, Deimos, Asteroids). I've restarted both my client and server since I encountered this yesterday, tried again tonight and it did the same thing.

![2017-05-16_23 12 22](https://cloud.githubusercontent.com/assets/720212/26137768/3b067b06-3a92-11e7-900d-2dfe2c7d3945.png)

**Comment by BlesseNtumble** [comment]
@radfast need you help)))
This bug is not only in my add-on.
In Amun-Ra, Extra Planet and other a similar problem.

**Comment by radfast** [comment]
Explain more please. 

1. Which exact Galacticraft version?

2. Only in SMP (you said "both client and server") or SSP also?

3. Please provide **fml-server-latest.log** with Debug messages = true in core.conf

4. Where is the Launch Controlled rocket launched from?

5. Do the launch and destination Launch Controllers definitely both have full energy bars?

6. Which tier rocket?  Does it happen with every tier rocket, e.g. Galacticraft's own Tier 3 Rocket?

7. Explain the picture because I don't know BlesseNtumble's mod - is that thing with the blue nosecone the rocket the player is supposed to be inside?  So the rocket is spawned, only in a different location from the player?

8. Can anyone confirm whether the same glitch happens for Moon / Mars **with no add-on installed**?

**Comment by radfast** [comment]
I found a possible bug in Galacticraft could be connected with this.  Please try new GalacticraftCore build 497.

**Comment by Fapiko** [comment]
@radfast Sorry for the late response, where can I get that build to try out?

**Comment by BlesseNtumble** [comment]
@Fapiko ci.micdoodle8.com

**Comment by radfast** [comment]
https://micdoodle8.com/mods/galacticraft/downloads  In the selection box select Minecraft version "1.7.10" then download the latest, that's now 499.

ci.micdoodle8.com is down right now, not sure why :/

**Comment by Fapiko** [comment]
Okay, updated to 499 and still getting this issue so let me answer your questions above:

1. Which exact Galacticraft version?
    - Was 486, but 499 still failing
1. Only in SMP (you said "both client and server") or SSP also?
    - Yes, tested in SSP and it happens in both.
1. Please provide fml-server-latest.log with Debug messages = true in core.conf
    - [fml-server-latest.log.txt](https://github.com/BlesseNtumble/GalaxySpace/files/1050092/fml-server-latest.log.txt)
1. Where is the Launch Controlled rocket launched from?
    - Overworld
1. Do the launch and destination Launch Controllers definitely both have full energy bars?
    - Yes
1. Which tier rocket? Does it happen with every tier rocket, e.g. Galacticraft's own Tier 3 Rocket?
    - I originally did it with the tier 3 rocket... tried the tier 4 rocket and it does the same thing.
1. Explain the picture because I don't know BlesseNtumble's mod - is that thing with the blue nosecone the rocket the player is supposed to be inside? So the rocket is spawned, only in a different location from the player?
    - This is the rocket - I entered creative mode and got out to take a screenshot. The player is in the rocket upon entry to Ceres but remains glitched out at the same altitude.
1. Can anyone confirm whether the same glitch happens for Moon / Mars with no add-on installed
    - Moon, Mars, Asteroids all work fine with add-ons (haven't tried without).

**Comment by BlesseNtumble** [comment]
https://youtu.be/L-v5W99OvYk

**Comment by Fapiko** [comment]
I noticed in @BlesseNtumble's video the rocket tier in Waila seems to be jumping around between 6 and something else - not sure if that gives any clues into the issue.

**Comment by BlesseNtumble** [comment]
There are just 2 rockets

**Comment by zzApotheosis** [comment]
Observe: https://drive.google.com/open?id=0B_wljXPJi507Wk96WDRaUlU3X1U

I went from Mars to Earth via Tier 3 Rocket. No problems.
I went from Earth to Mars via Tier 4 Rocket. No problems.
I went from Mars to Io via Tier 4 Rocket and encountered the bug.

So it seems that this bug happens when GalaxySpace dimensions are the destination. I have not tested the reverse; I don't know if this bug happens when a GalaxySpace dimension is the starting point and a Galacticraft dimension is the destination.
